src/modules/aws_vpc/entity/route.ts

```diff
@@ -0,0 +1,75 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../services/cloud-id';
+import { RouteTable } from './route_table';
+
+@Entity()
+export class Route {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => RouteTable, {
+    onDelete: 'CASCADE',
+    onUpdate: 'CASCADE',
+  })
+  @JoinColumn()
+  routeTable: RouteTable;
+
+  // destinations
+  @Column({ nullable: true })
+  @cloudId
+  DestinationCidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationIpv6CidrBlock?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  DestinationPrefixListId?: string;
+
+  // through
+  @Column({ nullable: true })
+  @cloudId
+  EgressOnlyInternetGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  GatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  InstanceId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  InstanceOwnerId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  NatGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  TransitGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  LocalGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  CarrierGatewayId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  NetworkInterfaceId?: string;
+
+  @Column({ nullable: true })
+  @cloudId
+  VpcPeeringConnectionId?: string;
+
+  @Column({ nullable: true })
```

Comment:
all fields are belonging to the cloud id? there is no single identifier for it?

Reply:
No unfortunately the Route entity in AWS doesn't have anything to identify it with. So we need to use a combination of all the fields to be able to identify it.